testsuite/integration-arquillian/pom.xml

```diff
@@ -575,4 +593,19 @@
         </profile>
     </profiles>
 
+    <repositories>
+        <repository>
+            <id>sonatype-snapshots</id>
+            <name>Sonatype Snapshots</name>
+            <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url>
+            <snapshots>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+            </snapshots>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+        </repository>
+    </repositories>
```

Comment:
You shouldn't need to define the repo here as it's defined in the keycloak-dependencies-admin-ui-wrapper. Have you tested without it here?

Reply:
I tried without it, and it says Could not find artifact org.keycloak:keycloak-admin-ui:jar:999-SNAPSHOT

Reply:
I'm okay with this for now as it's only introducing the Sonatype snapshot repo for the testsuite. We'll want to refactor something around this in the long run regardless, which me an @jonkoops can figure out somehow as it also relates to packaging of new account console + keycloak.js.

Reply:
Perhaps we can log an issue for this?

Reply:
#13468